tests/ci_build/Dockerfile.gpu

```diff
@@ -43,6 +43,9 @@ RUN set -ex; \
     chmod +x /usr/local/bin/gosu && \
     gosu nobody true
 
+RUN \
```

Comment:
@hcho3 I won't merge this PR any time soon. However, I found the NCCL version used for the CPP test is quite old and is from the base image.